docs/navigation-parameters-reference.md

```diff
@@ -1,295 +1,309 @@
+
+### items
+- **type**: array
+- **description**: an array of objects, each one being a link to a Luigi navigation node or an external URL. An item can have several attributes.
+- **attributes**:
+  - **label** defines the text for the link.
   - **testId** is a string where you can define your own custom `testId`. If nothing is specified, it is the node's label written as one word and lower case (e.g. `label`).
   - **icon** is the name of an icon from the [OpenUI](https://openui5.hana.ondemand.com/1.40.10/iconExplorer.html) or a custom link (relative or absolute) to an image displayed next to the label or instead of it.
   - **link** defines an absolute link to a **node**.
-  - **externalLink** is an object which indicates that the node links to an external URL. If this parameter is defined, the **link** parameter is ignored. It has the following properties:
-    - **sameWindow** defines if the external URL is opened in the current tab or in a new one. The default value for this parameter is `false`.
+  - **externalLink** is an object which indicates that the node links to an external URL. If this property is defined, the **link** property is ignored. It has the following attributes:
+    - **sameWindow** defines if the external URL is opened in the current tab or in a new one. The default value for this property is `false`.
     - **url** is the external URL that the link leads to.
 
 ## App switcher
 
 The app switcher is a drop-down list available in the top navigation bar. It allows you to switch between the navigation elements displayed in the drop-down. To do so, add the **appSwitcher** property to the **navigation** object using the following optional properties:
-- **showMainAppEntry** includes the link to the root of the Luigi application in the drop-down using the **title** specified in the **settings/header** section of the configuration as a label.
-- **items** defines the list of apps. App element properties are:
-  - **title** defines the app title. This is shown in the app switcher drop-down as well as the title in the header of the Luigi application if a user is in the context of the app. 
-  - **subTitle** defines the app sub-title. This is shown as the sub-title in the header of the Luigi application if a user is in the context of the app. 
-  - **link** is a link within the Luigi application that defines the root of the app. It is used to switch to the app if the drop-down entry is selected. It is also used to determine if a user is within the app's scope, so that the corresponding title and sub-title can be rendered in the header. 
+
+### showMainAppEntry
+- **type**: boolean
+- **description**: includes the link to the root of the Luigi application in the drop-down using the **title** specified in the **settings/header** section of the configuration as a label.
+
+### items
+- **type**: array
+- **description**: defines the list of apps. App element attributes are described below.
```

Comment:
Apps or applications? I'm fine with either one of them, just asking :)

Reply:
In the name **appSwitcher**, app stands for application. So applications is better, I think :)